triton/tut/serial.rst

```diff
@@ -2,195 +2,195 @@
-:doc:`previous tutorial <interactive>` showed how to
-run things directly from the command line, without any scripting needed.
+You learned, in the :doc:`interactive jobs <interactive>` 
+how all Triton users must do their computation by submitting jobs
+to the Slurm batch system to ensure efficient resource sharing. 
 
-Now let's see how to put these into scripts.  A **shell script** takes
-any commands that you might type directly into a shell and automates
-them.  The **slurm scripts** that we make in this lesson do do this.
-Scripts allow jobs to run asynchronously, in batch, and without human
-supervision.
+You additionally learned the interactive way to submit jobs,
+e.g. you could simply have an interative Bash session on a
+compute node. This proves useful for tests and debugging.
+Slurm jobs, however, are normally batch jobs, meaning that
+they are run unattended and asynchronously, without human 
+supervision. 
 
+To create a batch job, you need to create a job script and subsequently
+submit it to Slurm. A job script is simply a **shell script**, 
+e.g. Bash, where you put your **resource requests** and **job steps**. 
+You will see what these two components are shortly.
+You have already seen how to do these interactively; and in this tutorial
+you will learn how to bundle them in your job scripts.
 
+.. seealso::
 
-A basic script
-==============
-
-Let's say we want to run ``echo 'hello world'``. We have to tell the
-system how to run it.  Here is a simple submission script, put it in a
-file called ``hello.slrm`` (you can use the editor nano: ``nano
-hello.slrm``)::
+   Please refer to the :doc:`interactive jobs
+   <interactive>` tutorial to learn the basics
+   of Slurm.
 
-    #!/bin/bash
-    #SBATCH --time=0-00:05:00    # 5 mins
-    #SBATCH --mem-per-cpu=500    # 500MB of memory
+Your first job script
+=====================
 
-    srun echo 'hello, world'
+A job script is simply a shell script (Bash). And so the first line
+in the script should be the `shebang <https://en.wikipedia.org/wiki/Shebang_(Unix)>`_ directive (``#!``) followed by the
+full path to the executable binary of the shell's interpreter, which is 
+Bash in our case. What then follow are the resource requests and the job steps.
 
-**Whatever your application or programming language requires, you put
-it in the script.**
+Let's take a look at the following script 
 
-Each ``srun`` is a job step, and appears as a separate row in your
-history - which is useful for monitoring.  Then submit it with
-``sbatch``::
+.. code-block:: bash
 
-    $ sbatch hello.slrm
+   #!/bin/bash
+   #SBATCH --time=00:05:00    
+   #SBATCH --mem-per-cpu=100  
+   #SBATCH --output=/scratch/work/%u/hello.%j.out
+   #SBATCH --partition debug
 
-.. warning:: You must use ``sbatch``, not ``bash`` to submit the job
-   to process the ``#SBATCH`` headers and run in the background.
+   srun echo "Hello ${USER}! You are on node ${HOSTNAME}"
```

Comment:
The `${xxx}` works, but I wonder if $xxx would reduce confusion of a new person reading this (so they don't have to wonder why curly brackets are used here, but not other places).